lib/profile/commands/configure.rb

```diff
@@ -31,14 +31,31 @@ def display_details
       def ask_questions
         raise "No valid cluster types available" if !Type.all.any?
         type = cluster_type
+        smart_log = Logger.new(File.join(Config.log_dir,'configure.log'))
         @answers = prompt.collect do
           type.questions.each do |question|
             key(question.id).ask(question.text) do |q|
-              if type.fetch_answer(question.id)
-                q.default type.fetch_answer(question.id)
-              elsif question.default
-                q.default question.default
+
+              prefill = type.fetch_answer(question.id)
+              if question.default_smart
+                process = Flight::Subprocess::Local.new(
+                  env: {},
+                  logger: smart_log,
+                  timeout: 5,
+                )
+                result = process.run(question.default_smart, nil)
+                output = result.stdout.chomp
+                if !result.success?
+                  smart_log.debug("Command '#{question.default_smart}' failed to run: #{result.stderr}")
```

Comment:
I don't think that we need a logger here. I think that if the command doesn't return successfully, we should just fall back to the next default option.